conda/recipes/cudf/recipe.yaml

```diff
@@ -0,0 +1,127 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
```

Comment:
If you want, then this line can be ommitted as `recipe.yaml` is in the schema store now.

Reply:
done, thanks for the tip, @wolfv !